crates/sovereign-sdk/rollup-interface/src/node/services/da.rs

```diff
@@ -113,6 +115,19 @@ pub trait DaService: Send + Sync + 'static {
     /// Returns nothing if the transaction was successfully sent.
     async fn send_transaction(&self, blob: &[u8]) -> Result<Self::TransactionId, Self::Error>;
 
+    /// Send a transaction directly to the DA layer.
+    /// The default impl of this method actually blocks on `self.send_transaction`.
+    /// But the implementors can redefine it to be non-blocking.
+    async fn send_tx_no_wait(
```

Comment:
if we implement the function at trait level, why do we reimplement it in MockDa service.rs?

Reply:
1. Because DaService in general is not Clone, I can't do that trick with .clone & tokio::spawn here.
2. If there is no default impl then I have to implement it in every Da
3. I could implement it as `unimplemented` but then I would have to define it at least for MockDa?
4. So I made the most innocent default implementation that works for all cases and can be redefined. Indeed it was redefined for BitcoinService.